### PR TITLE
Update sites.yml

### PR DESCRIPTION
**Pull request by black-dinamo**
This PR is:

- [x] Adding a new domain
- [ ] Updating existing domain **size**
- [ ] Changing domain name
- [ ] Removing existing domain from list
- [ ] Website code changes (512kb.club site)
- [ ] Other not listed

- [x] I used the uncompressed size of the site
- [x] I have included a link to the GTMetrix report
- [x] The domain is in the correct alphabetical order
- [x] This site is not an ultra lightweight site
- [x] The following information is filled identical to the data file

***I confirm that I have read the [FAQ section](https://512kb.club/faq), particularly the two red items around minimal pages and inappropriate content and I attest that this site is neither of these things.***

- [x] Check to confirm

```
- domain: kaukokaipuu.xyz
  url: www.kaukokaipuu.xyz
  size:89.0kb
  last_checked: 2023-09-24
```

GTMetrix Report: https://gtmetrix.com/reports/www.kaukokaipuu.xyz/aTpZJxIf/#waterfall

### CODE REVIEW

**Comment by black-dinamo** [comment]
My bad, messed up the indentation at first.

File: _data/sites.yml [hunk 1]
```diff
@@ -1622,6 +1622,12 @@
   size: 3.0
   last_checked: 2022-04-26
 
+- domain: kaukokaipuu.xyz
+  url: https://www.kaukokaipuu
+  xyz/
```
Comment (garritfra) [inline _data/sites.yml:1627]: This still doesn't seem right. Do you mind fixing this?

**Review by garritfra — CHANGES_REQUESTED** [review]

File: _data/sites.yml (hunk 1 above)
Comment (black-dinamo) [reply _data/sites.yml]: Now I think I got It right. Thanks for the patience. First timer with Github here.

**Review by garritfra — APPROVED** [review]

**Comment by garritfra** [comment]
Thanks!

**Pull request merged**